packages/bunyan/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import { Axiom } from '@axiomhq/js';
+
+export enum AxiomEventLevel {
+    Trace = 'trace',
+    Debug = 'debug',
+    Info = 'info',
+    Warn = 'warn',
+    Error = 'error',
+    Fatal = 'fatal',
+    Silent = 'silent',
+}
+
+
+enum RecordLevel {
+    trace = 10,
+    debug = 20,
+    info = 30,
+    warn = 40,
+    error = 50,
+    fatal = 60,
+  }
+const noop = () => {};
+export class AxiomStream <T extends {msg: string, level: RecordLevel, time: Date }>  {
+    private axiom: Axiom;
+    dataset: string;
+    token: string;
+    orgId: string;
+    onError: (err: unknown) => void 
+
+    constructor(opt: Omit<AxiomStream<T>, 'write' |'onError'> & {
+        onError?: AxiomStream<T>['onError'];
+    }){
+        if (!opt.dataset){
+            throw new Error('dataset requried')
+        }
+
+        if (!opt.orgId){
+            throw new Error('orgId requried') 
+        }
+        
+        if (!opt.token){
+            throw new Error('token requried') 
+        }
+
+       this.orgId = opt.orgId;
+       this.token = opt.token
+       this.onError = opt.onError || noop
+       this.dataset = opt.dataset 
+
+       this.axiom = new Axiom({
+         orgId: this.orgId,
+         token: this.token
+       }) 
+    }
+
+   async write (record: Object) {
```

Comment:
how do we ensure that the Axiom events batch is flushed? e.g: `await axiom.flush()` is there a way to handle that from Bunyan?